test/integration/install_test.go

```diff
@@ -527,22 +527,31 @@ func TestInstallHelm(t *testing.T) {
 			"failed to generate root certificate for identity: %s", err)
 	}
 
-	var chartToInstall string
+	var baseChartToInstall string
+	var controlPlaneChartToInstall string
 	var vizChartToInstall string
 	var args []string
 	var vizArgs []string
 
 	if TestHelper.UpgradeHelmFromVersion() != "" {
-		chartToInstall = TestHelper.GetHelmStableChart()
+		baseChartToInstall = TestHelper.GetHelmStableChart()
 		vizChartToInstall = TestHelper.GetLinkerdVizHelmStableChart()
 		args, vizArgs = helmOverridesStable(helmTLSCerts)
 	} else {
-		chartToInstall = TestHelper.GetHelmChart()
+		baseChartToInstall = TestHelper.GetHelmCharts() + "/linkerd-base"
+		controlPlaneChartToInstall = TestHelper.GetHelmCharts() + "/linkerd-control-plane"
 		vizChartToInstall = TestHelper.GetLinkerdVizHelmChart()
 		args, vizArgs = helmOverridesEdge(helmTLSCerts)
 	}
 
-	if stdout, stderr, err := TestHelper.HelmInstall(chartToInstall, args...); err != nil {
+	releaseName := TestHelper.GetHelmReleaseName() + "-base"
+	if stdout, stderr, err := TestHelper.HelmInstall(baseChartToInstall, releaseName, args...); err != nil {
+		testutil.AnnotatedFatalf(t, "'helm install' command failed",
+			"'helm install' command failed\n%s\n%s", stdout, stderr)
+	}
+
+	releaseName = TestHelper.GetHelmReleaseName() + "-control-plane"
+	if stdout, stderr, err := TestHelper.HelmInstall(controlPlaneChartToInstall, releaseName, args...); err != nil {
```

Comment:
Wouldn't this fail with upgrade tests as `controlPlaneChartToInstall` would be empty here

Reply:
Yes, the upgrade integration tests have been disabled because the upgrade will be required to be manual. After the new charts are published we can refactor them, and turn them back on.